qa/integration-tests/src/test/java/io/camunda/it/migration/util/MigrationITInvocationProvider.java

```diff
@@ -0,0 +1,510 @@
+    };
+  }
+
+  private Map<String, String> tasklistElasticsearchDefaultConfig(final ZeebeUtil zeebe) {
+    return new HashMap<>() {
+      {
+        put("CAMUNDA_TASKLIST_ELASTICSEARCH_URL", "http://elasticsearch:9200");
+        put("CAMUNDA_TASKLIST_ELASTICSEARCH_HOST", "elasticsearch");
+        put("CAMUNDA_TASKLIST_ELASTICSEARCH_PORT", "9200");
+        put("CAMUNDA_TASKLIST_ZEEBEELASTICSEARCH_URL", "http://elasticsearch:9200");
+        put("CAMUNDA_TASKLIST_ZEEBE_GATEWAYADDRESS", zeebe.getZeebeGatewayAddress());
+        put("CAMUNDA_TASKLIST_ZEEBE_REST_ADDRESS", zeebe.getZeebeRestAddress());
+      }
+    };
+  }
+
+  private Map<String, String> tasklistOpensearchDefaultConfig(final ZeebeUtil zeebe) {
+    return new HashMap<>() {
+      {
+        put("CAMUNDA_TASKLIST_DATABASE", "opensearch");
+        put("CAMUNDA_TASKLIST_OPENSEARCH_URL", "http://opensearch:9200");
+        put("CAMUNDA_TASKLIST_OPENSEARCH_HOST", "opensearch");
+        put("CAMUNDA_TASKLIST_OPENSEARCH_PORT", "9200");
+        put("CAMUNDA_TASKLIST_ZEEBEOPENSEARCH_URL", "http://opensearch:9200");
+        put("CAMUNDA_TASKLIST_ZEEBE_GATEWAYADDRESS", zeebe.getZeebeGatewayAddress());
+        put("CAMUNDA_TASKLIST_ZEEBE_REST_ADDRESS", zeebe.getZeebeRestAddress());
+      }
+    };
+  }
+
+  private Stream<TasklistUtil.UserTaskArg> zeebeTasks(final DatabaseType db) {
+    return tasklistUtils.get(db).generatedTasks.get(TaskImplementation.ZEEBE_USER_TASK).stream();
+  }
+
+  public Stream<TasklistUtil.UserTaskArg> zeebeTasksForUpdate(final DatabaseType db) {
+    return tasklistUtils.get(db).generatedTasks.get(TaskImplementation.ZEEBE_USER_TASK).stream()
+        .filter(t -> t.apiVersion().equals(API_V2));
+  }
+
+  private Stream<TasklistUtil.UserTaskArg> jobWorkers(final DatabaseType db) {
+    return tasklistUtils.get(db).generatedTasks.get(TaskImplementation.JOB_WORKER).stream();
+  }
+
+  private void updateEnvMap(
+      final DatabaseType type,
+      final Map<DatabaseType, Map<String, String>> origin,
+      final Map<String, String> env) {
+    final var map = origin.getOrDefault(type, new HashMap<>());
+    map.putAll(env);
+    origin.put(type, map);
+  }
+
+  private void cloneProcesses(final DatabaseType type) throws IOException {
+    switch (type) {
+      case ELASTICSEARCH:
+        final var cfg = new ConnectConfiguration();
+        cfg.setUrl(elasticsearchExternalUrl);
+        cfg.setType("elasticsearch");
+        final var connector = new ElasticsearchConnector(cfg);
+        final var esClient = connector.createClient();
+
+        Awaitility.await()
+            .until(
+                () ->
+                    esClient
+                            .indices()
+                            .get(GetIndexRequest.of(req -> req.index("*")))
+                            .get("operate-process-8.3.0_")
+                        != null);
+
+        // Copy previous tasklist-process to operate-process, required for V2 APIs
+        esClient.reindex(
```

Comment:
😀 ❓ Is this the migration? Maybe we can integration our migration app, here? Like enabling it on 8.7?

Reply:
This is required because V2 APIs check against `operate-process` which, if you run only Tasklist 8.6 is not being populated. Spinning operate in the background would not require this _migration_ of processes.